documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -18,14 +18,15 @@ The `must-gather` tool collects the following logs and CR files in an archive fi
 ** `DataVolume` CR: Represents a disk mounted on a migrated VM.
 ** `VirtualMachine` CR: Represents a migrated VM.
 ** `Plan` CR: Defines the VMs and storage and network mapping.
+** `Job` CR: Optional: Represents a pre-migration hook, a post-migration hook, or both.
 
```

Comment:
@ibragins Line 21 OK? Our style guide says to avoid and/or. It wants "A or B,"  "A and B,"  or "A, B, or both." I suggest the last one.